octobot_trading/signals/signal_builder.py

```diff
@@ -0,0 +1,143 @@
+#  Drakkar-Software OctoBot-Trading
+#  Copyright (c) Drakkar-Software, All rights reserved.
+#
+#  This library is free software; you can redistribute it and/or
+#  modify it under the terms of the GNU Lesser General Public
+#  License as published by the Free Software Foundation; either
+#  version 3.0 of the License, or (at your option) any later version.
+#
+#  This library is distributed in the hope that it will be useful,
+#  but WITHOUT ANY WARRANTY; without even the implied warranty of
+#  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+#  Lesser General Public License for more details.
+#
+#  You should have received a copy of the GNU Lesser General Public
+#  License along with this library.
+import octobot_trading.signals.trading_signal as trading_signal
+import octobot_trading.signals.util as signal_util
+import octobot_trading.enums as trading_enums
+import octobot_trading.errors as trading_errors
+import octobot_trading.constants as trading_constants
+
+
+class SignalBuilder:
```

Comment:
👍

Reply:
A builder should contains a `build` method, do we need it?

Reply:
Right, I added it